src/content/docs/supporters/get-started.mdx

```diff
@@ -0,0 +1,223 @@
+      :::note
+      You may experience a delay before you can fund your wallet while your provider performs any necessary regulatory checks.
+      :::
+  2. Find and copy your wallet address or payment pointer. You'll need it for Step 4. Your wallet provider will assign one to you during sign up. The exact naming depends on what your provider has chosen to use.<br/><img src='/img/docs/extension/onboarding-2.png' alt='The Interledger Wallet dashboard displaying a wallet address' style='max-width:400px' class='img-outline'/>
+      <Tabs>
+      <TabItem label='Example wallet address'>
+        ``` 
+        https://wallet.example.com/alice
+        ```
+      </TabItem>
+      <TabItem label='Example payment pointer'>
+        ```
+        $wallet.example.com/alice
+        ```
+      </TabItem>
+      </Tabs>
+  3. Pin the extension to your toolbar (optional).<br/><img src='/img/docs/extension/onboarding-3.png' alt='An example of where to find the pin to toolbar option for the extension' style='max-width:300px' class='img-outline'/>
+  4. Enter the wallet address or payment pointer you found in Step 2.<br/><img src='/img/docs/extension/onboarding-4a.png' alt='Set up screen with fields for wallet address/payment pointer and amount and a renew monthly toggle (default is off).' style='max-width:300px' class='img-outline'/>
+  5. Enter the amount to make available for sending Web Monetization payments. If you choose not to renew monthly, you'll need to manually [add more funds](#adjust-your-budget) when your balance is zero.
+      :::note
+      The amount and renewal frequency is specific to the extension you're currently setting up. If you install the extension on a different browser and connect to the same wallet, that extension will have a separate budget. Payments made from one budget do not affect the balance of the other.
+      :::
+  6. Click **Connect**, then click **Agree** to allow the extension to connect to your wallet and add a key to your account.<br/><img src='/img/docs/extension/onboarding-4b.png' alt='Consent screen for allowing the extension to connect to your wallet with agree and decline buttons' style='max-width:300px' class='img-outline'/>
+      :::note
+      If your wallet provider's website opens in your browser, proceed to the next step. If you receive a connection error, review the instructions for [resolving the error](#resolve-a-key-addition-failure).
+      :::
+  7. Sign in to your wallet account if you haven't already, then confirm the connection and budget amount. For example:<br /><img src='/img/docs/extension/example-grant.png' alt='Example budget confirmation screen with accept and decline buttons' style='max-width:400px' class='img-outline'/>A success screen appears when the connection is successful.<br /><img src='/img/docs/extension/connection-success.png' alt='Web page displaying a connection successful screen' style='max-width:400px' class='img-outline'/>You're now ready to set up continuous payments.
```

Comment:
Should we move the "success screen" part a separate step? This step looks bit large due to two images:
<img alt="" src="https://github.com/user-attachments/assets/6fc48066-7074-4c32-892a-e489f1430e0c" width="300">